src/test/java/com/segment/analytics/android/integration/firebase/FirebaseTest.java

```diff
@@ -0,0 +1,5 @@
+package com.segment.analytics.android.integration.firebase;
+
+/* TO DO:
+ * Add Android test support
```

Comment:
Needs tests...

Reply:
Spoke with Joe and Brittany, and we're going to delay writing tests for the moment to get the beta out, but tests are priority once I get more practice with our test suite.

Reply:
Got it.  I do think there's a lot of room for bugs to slip through here and would be much more confident with the code if we had tests. But, if we can accept the risk that sounds ok for now.